### PR TITLE
Update documentation and FAQ.

### PR DESCRIPTION
**Pull request by oliverchang**
Also point to OSSF OSV schema docs.

### CODE REVIEW

**Comment by oliverchang** [comment]
friendly ping!

File: lib/osv/vulnerability.proto [hunk 1]
```diff
@@ -55,13 +55,9 @@ message Package {
...
-  // Required. The ecosystem for this package. For vulnerabilities from OSS-Fuzz
-  // that do not have a package ecosystem (e.g. C/C++ packages), this is
-  // "OSS-Fuzz".
-  // Other valid values are:
-  //   - "Go" for Go modules.
-  //   - "PyPI" for Python packages.
-  //   - "npm" for NPM packages.
+  // Required. The ecosystem for this package. 
+  // For the complete list of valid ecosystem names, see
+  // <https://ossf.github.io/osv-schema/#affectedpackage-field>.
```
Comment (jonathanmetzman) [inline lib/osv/vulnerability.proto:60]: Why the brackets?

File: docs/faq.md [hunk 2]
```diff
@@ -19,83 +20,39 @@ OSV can be used by both:
...
 
 ## Where does the data come from?
 
-Currently, our data aggregates [thousands of vulnerabilities](https://bugs.chromium.org/p/oss-fuzz/issues/list?q=Type%3DBug-Security%20-status%3AWontFix%2CDuplicate&can=1) found via fuzzing
-[380+ open source projects](https://github.com/google/oss-fuzz/tree/master/projects)
-integrated with
-[OSS-Fuzz](https://github.com/google/oss-fuzz). We are planning to work with the
-open source community to
-[expand this to various package ecosystems](https://github.com/google/osv/issues/44).
+Please see the above "What is OSV" section!
```
Comment (jonathanmetzman) [inline docs/faq.md:37]: nit: You should probably link to it.

**Review by jonathanmetzman — APPROVED** [review]
lgtm

File: lib/osv/vulnerability.proto (hunk 1 above)
Comment (oliverchang) [reply lib/osv/vulnerability.proto:60]: This is part of the Markdown spec for explicitly writing links: https://daringfireball.net/projects/markdown/syntax#autolink

File: docs/faq.md (hunk 2 above)
Comment (oliverchang) [reply docs/faq.md]: Good point, done!

**Pull request merged**